seekpath/getpaths.py

```diff
@@ -325,3 +325,268 @@ def get_explicit_k_path(
     for k, v in explicit_res.items():
         res["explicit_{}".format(k)] = v
     return res
+
+
+def get_path_orig_cell(
+    structure,
+    with_time_reversal=True,
+    recipe="hpkot",
+    threshold=1.0e-7,
+    symprec=1e-05,
+    angle_tolerance=-1.0,
+):
+    r"""
+    Return the kpoint path information for band structure given a
+    crystal structure, using the paths from the chosen recipe/reference.
+    The original unit cell is used. Standardization or symmetrization of the
+    input unit cell is not performed.
+
+    If the provided unit cell is a supercell of a smaller primitive cell,
+    return the standard k path of the smaller primitive cell in the basis
+    of the supercell reciprocal lattice vectors. In this case, the k point
+    labels lose their meaning: they are not at the high-symmetry points of
+    the first BZ of the given supercell.
```

Comment:
```suggestion
    of the supercell reciprocal lattice vectors. Note that, in this case,
    the k-point labels are not at the high-symmetry points of
    the first BZ of the given supercell.
```

Reply:
I did a slightly modified version: "In this case, the k-point labels lose their meaning as the corresponding k-points are not at the high-symmetry"